AwesomeProject/screens/AboutMe/AboutMe.screen.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {View, Text, ScrollView} from 'react-native';
+import styles from './AboutMe.styles';
+import BreakLine from '../../components/breakline';
+
+const AboutMe = () => {
+  return (
+    <View style={styles.container}>
+      <ScrollView>
+        <View style={styles.header}>
+          <Text style={styles.boldText}>Alexey Kundro</Text>
+        </View>
+        <View style={styles.main}>
+          <Text style={styles.mainText}>
+            Let me introduce myself.<BreakLine /> My full name is Alexey Kundro. I'm 23. I
```

Comment:
I  understood. You can't do it like that.
Better to do:
`
<Text> Let me introduce myself</Text>
<Text> My full name ... </Text>
<Text> Contacts: exey.kundro@gmail.com </Text>
`

Reply:
delete <BreakLine component please :)

Reply:
Fixed